pcap_to_node_pcap/Dockerfile

```diff
@@ -13,6 +13,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     make \
     gcc \
     g++ \
+    tshark \
```

Comment:
Looks like tshark doesn't respect `-y`:
```
debconf: falling back to frontend: Readline
Configuring wireshark-common
----------------------------

Dumpcap can be installed in a way that allows members of the "wireshark" system
group to capture packets. This is recommended over the alternative of running
Wireshark/Tshark directly as root, because less of the code will run with
elevated privileges.

For more detailed information please see
/usr/share/doc/wireshark-common/README.Debian.

Enabling this feature may be a security risk, so it is disabled by default. If
in doubt, it is suggested to leave it disabled.

Should non-superusers be able to capture packets? [yes/no] 
```